.deployment.config.json

```diff
@@ -6,10 +6,10 @@
       "sequential": ["FIRST_MAIN_REGION"]
     },
     "environments_order": {
-      "sequential": $[ENVIRONMENTS]
+      "sequential": ["dev", "stg", "prd"]
     },
     "team_jenkins": "searchuibuilds",
-    "start_environment_automatically": false,
+    "start_environment_automatically": true,
```

Comment:
What does this do? Why is this change required?

Reply:
It's the other way around, we no longer need this to be `false`.

Having this set to `false` meant that we were getting prompted before a package would progress from `dev` to `stg` and `stg` to `prd`. However, to accept it, we needed to connect to some of our back-end infrastructure, which could take some time. Now, since releases are manually triggered by a maintainer, we don't need this.